### ISSUE TITLE
Corneal dellen

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10937]
Preferred term label:
Corneal dellen

Synonyms
Corneal thinning, Localized corneal thinning, Corneal stromal dellen

Definition (free text, please give PubMed ID)
A localized area of corneal thinning typically occurring adjacent to an elevated lesion, such as a conjunctival scar, pterygium, or poorly fitting contact lens. Corneal dellen result from tear film instability and localized dehydration. Without timely treatment, they may progress to epithelial defects or corneal ulceration.

Parent term (use hpo.jax.org/app)
Decreased corneal thickness (HP:100689)

Diseases characterized by this term
Pterygium, Corneal ulceration , Post-surgical ocular surface disorder

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001192
- label: Corneal dellen
- parents: [Decreased corneal thickness (HP:0100689)](https://hpo.jax.org/app/browse/term/HP:0100689)
- pmid: PMID:32596070
- def: A localized area of corneal thinning typically occurring adjacent to an elevated lesion, such as a conjunctival scar, pterygium, or poorly fitting contact lens. Corneal dellen result from tear film instability and localized dehydration. Without timely treatment, they may progress to epithelial defects or corneal ulceration.
- comment: Diseases characterized by this finding include corneal pterygium, corneal ulceration, and post-surgical ocular surface disorder.
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10937
- synonyms: Corneal stromal dellen(NONE)|Localized corneal thinning(NONE)
